lib/api/src/js/module.rs

```diff
@@ -466,10 +528,13 @@ impl Module {
             return Err("The exports length must match the type hints lenght".to_owned());
         }
         for (i, val) in exports.iter().enumerate() {
-            let kind = Reflect::get(val.as_ref(), &"kind".into())
-                .unwrap()
-                .as_string()
-                .unwrap();
+            #[allow(unused_unsafe)]
+            let kind = unsafe {
```

Comment:
Why we are doing this? `#[allow(unused_unsafe)]` and then `unsafe {`. Doesn't make that much sense.
Perhaps I'm missing something?

Reply:
I think this is due to js bindgen compilation quirks.

Some functions apparently are unsafe when compiled to wasm32-, but aren't unsafe when the target is native Rust.

So this prevents compilation warnings/errors when doing `cargo check` without `--target wasm32-*`.

Reply:
That's weird, because `Reflect::get` is a safe function for all targets (thus usage should also be safe).
https://rustwasm.github.io/wasm-bindgen/api/js_sys/Reflect/fn.get.html

I don't think this is really necessary and it bloats a bit the code?

Reply:
I'm not sure where it comes in, exactly, but I did try to remove the annotation and then had "requires unsafe" errors.